jmx-metrics/src/main/resources/target-systems/cassandra.groovy

```diff
@@ -149,3 +100,44 @@ otel.instrument(compactionCompletedTasks,
         "cassandra.compaction.tasks.completed",
         "Number of completed compactions since server [re]start", "1", "Value",
         otel.&longCounterCallback)
+
+
+def clientRequests = otel.mbeans([
+  "${clientRequestRangeSlice},name=Latency",
+  "${clientRequestRangeSlice},name=Unavailables",
+  "${clientRequestRangeSlice},name=Timeouts",
+  "${clientRequestRangeSlice},name=Failures",
+  "${clientRequestRead},name=Latency",
+  "${clientRequestRead},name=Unavailables",
+  "${clientRequestRead},name=Timeouts",
+  "${clientRequestRead},name=Failures",
+  "${clientRequestWrite},name=Latency",
+  "${clientRequestWrite},name=Unavailables",
+  "${clientRequestWrite},name=Timeouts",
+  "${clientRequestWrite},name=Failures",
+])
+
+otel.instrument(clientRequests,
+  "cassandra.client.request.count",
+  "Number of requests",
```

Comment:
Some specificity in the names and descriptions is being lost.  Is it potentially clearer to batch each scope and just have a status attribute?